utility/README.md

```diff
@@ -1,679 +1,1309 @@
 
+Non-profit seeking seeking attacks/behaviors which cost more to perform than can reasonably be expected to benefit the attacker. Usually results in damage to the network's reputation.
 
+**AKA**: Kill 'em all.
 
-* <span style="text-decoration:underline;">Mad Man Lazy</span>.  An attack which does not require significant time, energy or resources.  AKA:  What happens when I push this button?
-* <span style="text-decoration:underline;">Mad Man Hacker:</span> A non-profit seeking  attack which requires creation, modification and upkeep of the core software in a private repository.
-* <span style="text-decoration:underline;">Masochistic Mad Man</span>: Any attack which, although possible, makes no sense because you could accomplish the same thing without doing all that work.
+##### 5.1.8.1 Mad Man Lazy
 
+Attack does not require significant time, energy or resources.
 
-## 5.2 Specifics
+**AKA**: What happens when I push this button?
 
-**Fisherman collusion with ServiceNodes (Profit seeking passive, Collusion active, Hacker)**
+##### 5.1.8.2 Mad Man Hacker
 
-If a Fisherman is colluding with one or more ServiceNodes and is in possession of the private key(s) of those nodes, he is able to falsify all aspects of that node’s report card.  Therefore, all of his colluding node partners get A+ report cards and resultantly larger paychecks.
+Non-profit seeking attack which requires creation, modification and upkeep of the core software in a private repository.
 
-This is the “big one”.  It is the primary reason that Fishermen are (at this stage) DAO Approval is required.  It is why off chain data logs, good-citizen reporting and DAO oversight exist. It is also the reason that Fishermen require large stake/deposit, as well as increased destaking period and delayed payments  (See attached spreadsheet of projected collusion ROI based on such factors as Node Percentage, Risk Rate, etc.)
+##### 5.1.8.3 Masochistic Mad Man
+
+Any attack which, although possible, makes no sense because you could accomplish the same thing without doing all that work.
+
+**AKA**: Rube Goldberg Attack.
+
+### 5.2 Attack Examples
+
+<!-- TODO(olshansky): Review & Update -->
+
+#### 5.2.1 Fisherman <> Servicer Collusion
+
+**Attack Vectors**: Profit Seeking Passive; Collusion Active; Hacker
+
+A Fisherman colluding with one or more Servicers and is in possession of their keys is able to falsify all aspects of that Servicer's report card. Therefore, all of his colluding node partners get A+ report cards and resultantly larger paychecks.
+
+This is the “big one”. It is the primary reason that Fishermen are (at this stage) DAO Approval is required. It is why off chain data logs, good-citizen reporting and DAO oversight exist. It is also the reason that Fishermen require large stake/deposit, as well as increased destaking period and delayed payments (See attached spreadsheet of projected collusion ROI based on such factors as Node Percentage, Risk Rate, etc.)
```

Comment:
`It is the primary reason that Fisherman (at this stage) require DAO Approval.`
Sounds better here